internal/runtime/client/client.go

```diff
@@ -375,6 +376,11 @@ func httpCall(ctx context.Context, request, response runtime.Object, opts *httpC
 		return errors.Wrapf(err, "failed to compute URL of the extension handler %q", opts.name)
 	}
 
+	// Create request latency metric.
+	start := time.Now()
+	defer func() {
+		runtimemetrics.RequestLatency.Observe(opts.hookGVH.Hook, *extensionURL, time.Since(start))
```

Comment:
Q: should we use the full gvh in some form as label? Please check how Kubernetes does this in the apiserver for it's rest endpoint metrics

Reply:
Sounds good